manifests/init.pp

```diff
@@ -66,6 +75,7 @@
   } ->
   ffnord::fastd { "fastd_${mesh_code}":
     mesh_code       => $mesh_code,
+    batman_code     => $mesh_code,
```

Comment:
Why do we need an extra parameter batman_code, when it is the same as mesh_code, in all applied situations?